src/utils/markdown/tabs/tabs-script.ts

```diff
@@ -45,15 +45,15 @@ export const enableTabs = () => {
 			tab.focus();
 			tab.click();
 
-			// Scroll onto screen in order to avoid jumping page locations
-			setTimeout(() => {
-				tab.scrollIntoView &&
-					tab.scrollIntoView({
-						behavior: "auto",
-						block: "center",
-						inline: "center",
-					});
-			}, 0);
+			// Redundant: Scroll handled by `handleClick` on `tab.click()`
+			// setTimeout(() => {
+			// 	tab.scrollIntoView &&
+			// 		tab.scrollIntoView({
+			// 			behavior: "auto",
+			// 			block: "center",
+			// 			inline: "center",
+			// 		});
+			// }, 0);
```

Comment:
This is still needed because keyboard interaction will not necessarily keep the tabs in view; we should ensure that the buttons are visible here even if `shouldScrollToTab` is false.

Unfortunately, we can't rely on `tab.focus()` or the `block: "nearest"` option for `scrollIntoView()`, as it does not take the page's sticky header into account. The tab buttons can still be focused and concealed behind the header, because the browser only checks that they are within the viewport.

If you're up for it, I do think this could be improved with the following:
- Replace`tab.click()` with a `changeTabs(tabName)` call (so that it never redundantly runs the `window.scroll` in `handleClick`)
- Uncomment the following `setTimeout` / `tab.scrollIntoView`
- Add a comment to document why this is done (tbh I should've done that last time I touched this, so this is my bad)

Reply:
I've replaced `tab.click()` with `changeTabs(tabName)` and also replaced the deprecated `e.keyCode` with `e.code`. 

For the scroll, I've added some code to preserve the scroll location if the tabs are within the viewport, else it will scroll to 20px below the header. Let me know if this is preferred, might also be worth implementing this on the table of contents so it scrolls to 20px below the header instead of center?

Reply:
@fennifith @bk7312 I can test this on my mac when I get a moment later.